.circleci/config.yml

```diff
@@ -43,7 +67,8 @@ jobs:
           at: *workspace-dir
       - sonarcloud/scan:
           sonar_token_variable_name: SONAR_TOKEN
-  test_live:
+
+  test_live_sonar:
```

Comment:
because we are testing and running sonar as well?

Reply:
Why? This job doesn't run sonar

Reply:
then why are we renaming the job name?

Reply:
No, it was just a name change. It is because it threw a warning that the test_live job was already defined. I will name it more appropriately.